### PR TITLE
Clarify statement at the cmd wizard tool, for new users

### PR DESCRIPTION
**Pull request by CarlosGS**
# What does this implement/fix? 

The command line wizard ends with "If all goes well, you should see your ESP being discovered automatically.".

But this is not the case, it just generates the yaml config file; my edit is to clarify that.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other

**Related issue or feature (if applicable):** fixes <link to issue>

**Pull request in [esphome-docs](https://github.com/esphome/esphome-docs) with documentation (if applicable):** esphome/esphome-docs#<esphome-docs PR number goes here>

## Test Environment

- [ ] ESP32
- [ ] ESP32 IDF
- [ ] ESP8266

## Example entry for `config.yaml`:

```yaml
# Example config.yaml

```

## Checklist:
  - [ ] The code change is tested and works locally.
  - [ ] Tests have been added to verify that the new code works (under `tests/` folder).
  
If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [esphome-docs](https://github.com/esphome/esphome-docs).

### CODE REVIEW

**Review by davet2001 — APPROVED** [review]
Looks good!

File: esphome/wizard.py [hunk 1]
```diff
@@ -367,10 +367,7 @@ def wizard(path):
     )
     safe_print()
     safe_print("Next steps:")
-    safe_print(
-        '  > Check your Home Assistant "integrations" screen. If all goes well, you '
-        "should see your ESP being discovered automatically."
-    )
-    safe_print("  > Then follow the rest of the getting started guide:")
+    safe_print("  > Follow the rest of the getting started guide:")
     safe_print("  > https://esphome.io/guides/getting_started_command_line.html")
```
Comment (oxan) [inline esphome/wizard.py]: Let's link to the section where we are:
```suggestion
    safe_print("  > https://esphome.io/guides/getting_started_command_line.html#adding-some-features")
```

**Review by oxan — APPROVED** [review]

**Comment by CarlosGS** [comment]
Thanks @oxan! Wanted to link there but wasn't sure of the formatting the test was requesting :D

**Comment by oxan** [comment]
We use formatting as enforced by [black](https://black.readthedocs.io/en/stable/) ;)

**Pull request merged**